orchestrator/services/processes.py

```diff
@@ -356,7 +375,11 @@ def create_process(
         raise
 
     pstat = ProcessStat(
-        pid, workflow=workflow, state=Success({**state, **initial_state}), log=workflow.steps, current_user=user
+        pid,
+        workflow=workflow,
+        state=Success({**state, **initial_state}),
```

Comment:
I think you can now do

`state=Success(state | Inititial_state}`

Probably in other places as well